README.md

```diff
@@ -28,3 +28,20 @@ at your option.
 Unless you explicitly state otherwise, any contribution intentionally submitted
 for inclusion in the work by you, as defined in the Apache-2.0 license, shall be
 dual licensed as above, without any additional terms or conditions.
+
+### Run integration tests
```

Comment:
I wonder if we should start a "contributor guidelines" file (usually `CONTRIBUTING.md`). GitHub has some more details about that here if you're curious: https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors

Don't have to do that as part of this PR though.

Reply:
Thanks, I will look into that!